doc/reference/reference_lua/datetime.rst

```diff
@@ -1,62 +1,955 @@
+..  _datetime-module:
 
 Module datetime
 ===============
 
-Since :doc:`2.10.0 </release/2.10.0>`.
+Since: :doc:`2.10.0 </release/2.10.0>`
 
-The ``datetime`` module provides support for the :ref:`datetime and interval data types <index-box_datetime>`.
+The ``datetime`` module provides support for the :ref:`datetime <index-box_datetime>`and :ref:`interval <index-box_interval>` data types.
 It allows creating the date and time values either via the object interface
 or via parsing string values conforming to the ISO-8601 standard.
 
-Below is a list of the ``datetime`` module functions and methods.
 
-.. container:: table
+.. _uri-module-api-reference:
+
+API Reference
+-------------
+
+Below is a list of ``datetime`` functions, properties, and related objects.
+
+..  container:: table
 
     ..  rst-class:: left-align-column-1
     ..  rst-class:: left-align-column-2
 
     ..  list-table::
-        :widths: 25 75
-        :header-rows: 1
+        :widths: 35 65
+
+        *   -   **Functions**
+            -
+
+        *   -   :ref:`datetime.new() <datetime-new>`
+            -   Create an object of the ``datetime`` type from a table of time units
+
+        *   -   :ref:`datetime.now() <datetime-now>`
+            -   Create an object of the ``datetime`` type with the current date and time
+
+        *   -   :ref:`datetime.is_datetime() <datetime-is_datetime>`
+            -   Check whether the specified value is a ``datetime`` object
+
+        *   -   :ref:`datetime.parse() <datetime-parse>`
+            -   Convert an input string with the date and time information into a ``datetime`` object
+
+        *   -   :ref:`datetime.interval.is_interval() <datetime-interval-is_interval>`
+            -   Check whether the specified value is an ``interval`` object
+
+        *   -   :ref:`datetime.interval.new() <datetime-interval-new>`
+            -   Create an object of the ``interval`` type from a table of time units
+
+        *   -   **Properties**
+            -
+
+        *   -   :ref:`datetime.TZ <datetime-tz>`
+            -   An array of timezone names and abbreviations
+
+        *   -   **Related objects**
+            -
+
+        *   -   :ref:`datetime_object <datetime_obj>`
```

Comment:
I propose to add a subtable with datetime-related methods: `totable`, `format`, `set`, `add`, `sub` with links to the references.

Reply:
Nice, thanks!